### PR TITLE
Adding Alert Rule Notification Email schema

### PR DESCRIPTION
**Pull request by stephoule76**

### CODE REVIEW

**Comment by johntdyer** [comment]
@stephoule76  pretty sure you need to bump the version of go-thousandeyes to pick up your change 

example: https://github.com/william20111/terraform-provider-thousandeyes/pull/55/files

File: go.mod [hunk 1]
```diff
@@ -19,5 +19,5 @@ require (
 	github.com/posener/complete v1.2.3 // indirect
 	github.com/ulikunitz/xz v0.5.10 // indirect
 	github.com/vmihailenco/msgpack v4.0.4+incompatible // indirect
-	github.com/william20111/go-thousandeyes v1.2.1
+	github.com/william20111/go-thousandeyes v1.2.2
```
Comment (william20111) [inline go.mod:22]: you will want to bump to v1.3.0

**Review by william20111 — COMMENTED** [review]
bump to latest

File: go.mod (hunk 1 above)
Comment (stephoule76) [reply go.mod]: Done, updated to 1.3.0.

**Pull request merged**